lib/managers/files.js

```diff
@@ -576,6 +576,27 @@ Files.prototype.deletePermanently = function(fileID, callback) {
 };
 
 /**
+ * Retrieves all of the tasks for given file.
+ *
+ * API Endpoint: '/files/:fileID/tasks'
+ * Method: GET
+ *
+ * @param {string} fileID - The ID of the file to get tasks for
+ * @param {?Object} qs - Additional options can be passed with the request via querystring. Can be left null in most cases.
+ * @param {Function} callback - Called with nothing if successful, error otherwise
```

Comment:
This comment needs to be updated; the callback is called with the tasks when successful